msal/consumer-rules.pro

```diff
@@ -17,7 +17,7 @@
 #}
 
 ##---------------Begin: proguard configuration for MSAL  --------
--keep class com.microsoft.** { *; }
+-keep class com.microsoft.identity.client.** { *; }
```

Comment:
I think this should be com.microsoft.identity.**
Is it good enough for your app?